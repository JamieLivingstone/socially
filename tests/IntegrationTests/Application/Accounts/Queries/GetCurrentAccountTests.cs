using System.Threading.Tasks;
using Application.Accounts.Queries.GetCurrentAccount;
using IntegrationTests.Application.TestUtils;
using NUnit.Framework;
using Snapshooter.NUnit;

namespace IntegrationTests.Application.Accounts.Queries;

[TestFixture]
public class GetCurrentAccountTests : TestBase
{
  [Test]
  public async Task GivenAnAuthenticatedRequest_ReturnsAccount()
  {
    var query = new GetCurrentAccountQuery();

    var account = await SendAsync(query);

    Snapshot.Match(account);
  }
}
