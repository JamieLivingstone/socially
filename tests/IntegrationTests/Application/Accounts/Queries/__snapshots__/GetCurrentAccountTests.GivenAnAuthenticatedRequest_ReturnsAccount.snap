{
  "Name": "Liam Hale",
  "Username": "liam",
  "Email": "liam.hale@test.com"
}
